.eslintrc.js

```diff
@@ -139,7 +139,7 @@ module.exports = {
   ],
   ignorePatterns: [
     '**/types/**/*',
-    '**/test/**/fixtures/**/*',
+    '**/test/**/fixture*/**/*',
```

Comment:
:sob:

Reply:
I am trying to stop using plurals in dir names since it seems redundant to me